### ISSUE TITLE
RocketChat KT questions

### ISSUE THREAD

**Issue by ShellyXueHan** [bcgov/rocketchat#11]
### devops:
- is tekton currently in use? doesn't look complete tho
- i've got some change on the github workflow included in PR: https://github.com/bcgov/rocketchat/pull/10. Not sure if we were to switch over to tekton later on?
- how does cross-cluster disaster recovery work: what should be the correct configuration for `TransportServer`?
- how is the RC image build in artifactory? what does an update process look like?
- what does `data-pump` pod do? When do I need to run them?
- `reggie/openshift` not needed right?
- how do i get vault access?
- is `rocketchat-testplan.jmx` also in use?

**Comment by jleach** [comment]
> ### devops:
> * is tekton currently in use? doesn't look complete tho

No. While I think Tekton is great I don't see any advantage over GitHub Actions except you can grant less access to a SA (the one used by GitHub Actions). 

> * i've got some change on the github workflow included in PR: [Reggie web - add an email hint #10](https://github.com/bcgov/rocketchat/pull/10). Not sure if we were to switch over to tekton later on?
> * how does cross-cluster disaster recovery work: what should be the correct configuration for `TransportServer`?

Yes. We will need mongo replication between clusters. When its time to set this up LMK and I can share what I know. There are some finer points to mongoDB replication.

> * how is the RC image build in artifactory? what does an update process look like?

Not build. Artifactory just pulls the image from DockerHub. We don't modify the image at all.

> * what does `data-pump` pod do? When do I need to run them?

I used that form migration. It can be deleted. I just spun up a pod with mongo CLI in it so I could restore a backup from a PVC. 

> * `reggie/openshift` not needed right?

I don't think so. 

> * how do i get vault access?

Justin. When its ready I'll do some more KT and show you how it all fits together.

> * is `rocketchat-testplan.jmx` also in use?

Not sure what this file is.
